### PR TITLE
Add ELK Stack.

### PR DESCRIPTION
**Pull request by andreiavrammsd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # docker-compose collection
 
-### Contents
+## Contents
```
Comment (cristaloleg) [inline README.md:3]: make sense

File: README.md [hunk 2]
```diff
@@ -340,3 +341,28 @@ services:
...
+### Search
+
+- [ELK Stack: Elasticsearch, Logstash, Kibana](#ELK)
+
+#### ELK
+```yaml
+
+# Required on Linux:
+# sudo sysctl -w vm.max_map_count=262144
+# sudo sh -c 'echo "vm.max_map_count=262144" >> /etc/sysctl.conf'
```
Comment (cristaloleg) [inline README.md:354]: what do you think to add a source(url) to this info?

File: README.md [hunk 3]
```diff
@@ -340,3 +341,28 @@ services:
...
+
+services:
+  elk:
+    image: sebp/elk:683
```
Comment (cristaloleg) [inline README.md:359]: Why not https://github.com/deviantony/docker-elk ?

Comment (andreiavrammsd) [reply README.md:359]: Never used it. I'm using sebp/elk in development since a few weeks ago and I'm happy with it.

**Pull request merged**